postgres-14/Dockerfile

```diff
@@ -12,16 +12,25 @@ LABEL maintainer="Citus Data https://citusdata.com" \
       org.label-schema.schema-version="1.0"
 
 ENV CITUS_VERSION ${VERSION}.citus-1
+#install postgresql
+RUN apt-get update  \
+    && apt-get install -y lsb-release \
+        apt-utils \
+        wget \
+        gnupg2  \
+    && apt-get clean all \
+    && wget --quiet -O - https://www.postgresql.org/media/keys/ACCC4CF8.asc | apt-key add - \
+    && echo "deb http://apt.postgresql.org/pub/repos/apt $(lsb_release -cs)-pgdg main 14" > /etc/apt/sources.list.d/pgdg.list \
+    && apt-get purge -y --auto-remove curl \
+    && rm -rf /var/lib/apt/lists/*
 
 # install Citus
 RUN apt-get update \
```

Comment:
It does not make sense to me that we have 2 separate `RUN` commands that start with `apt-get update`. Can we merge that into one?

Reply:
Fixed